src/shared/components/palette/dynamic-multi-select-list/write-dynamic-multi-select-list-field.html

```diff
@@ -0,0 +1,23 @@
+<div class="form-group bottom-30" [ngClass]="{'error': !checkboxes.valid && checkboxes.touched}" [id]="id()">
+
+    <fieldset>
+  
+      <legend>
+        <span class="form-label" *ngIf="caseField.label">{{caseField | ccdFieldLabel}}</span>
```

Comment:
Being finicky again but for readability can class="form-label" be moved after the ngIf as per the next two lines